lib/iban_calculator/active_support.rb

```diff
@@ -0,0 +1,31 @@
+# these are methods implemented in active_support 4 which
+# are required to run this gem with active_support 3
+require 'active_support/configurable'
+
+module IbanCalculator
+  include ActiveSupport::Configurable
+
+  # taken from: activesupport/lib/active_support/core_ext/hash/keys.rb
+  Hash.class_eval do
+    def deep_stringify_keys!
+      deep_transform_keys!{ |key| key.to_s }
+    end
+
+    def deep_transform_keys!(&block)
+      keys.each do |key|
+        value = delete(key)
+        self[yield(key)] = value.is_a?(Hash) ? value.deep_transform_keys!(&block) : value
+      end
+      self
+    end
+  end
+end
+
+class << IbanCalculator
+  alias_method :old_config_accessor, :config_accessor
```

Comment:
Sure. Rather than modify the gem to not use deep_stringify_keys! I chose to implement the method in the gem as the method was only introduced in active_support 4.
https://github.com/rails/rails/blob/4-0-stable/activesupport/lib/active_support/core_ext/hash/keys.rb

You can verify that these methods are missing from the 3-2-stable branch of active_support.
https://github.com/rails/rails/blob/3-2-stable/activesupport/lib/active_support/core_ext/hash/keys.rb 

With regards to the config_accessor method again have a look at the 3-2-stable branch.
https://github.com/rails/rails/blob/3-2-stable/activesupport/lib/active_support/configurable.rb#L54

Since 3-2's config_accessor doesn't take a block or provide an option to set the default value of a config I wrote this quick hack.